From e9f8daf7e1261d6936bebfb7d3097c32ab2f8779 Mon Sep 17 00:00:00 2001
From: Adrien Plazas <kekun.plazas@laposte.net>
Date: Thu, 14 Oct 2021 12:36:39 +0200
Subject: [PATCH] window: Reduce the minimum height

This way it can properly adapt to mobiles with a virtual keyboard.
---
 src/gui/gcal-window.ui | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gui/gcal-window.ui b/src/gui/gcal-window.ui
index c54ef0fd..36d08a18 100644
--- a/src/gui/gcal-window.ui
+++ b/src/gui/gcal-window.ui
@@ -5,7 +5,7 @@
     <property name="window_position">center</property>
     <property name="default_width">800</property>
     <property name="default_height">600</property>
-    <property name="height_request">600</property>
+    <property name="height_request">200</property>
     <property name="show_menubar">False</property>
     <signal name="window-state-event" handler="window_state_changed" object="GcalWindow" swapped="no"/>
     <style>
-- 
2.30.2

